sky/clouds/aws.py

```diff
@@ -425,7 +425,7 @@ def make_deploy_resources_variables(self,
 
     def _get_feasible_launchable_resources(
         self, resources: 'resources_lib.Resources'
-    ) -> Tuple[List['resources_lib.Resources'], List[str]]:
+    ) -> Tuple[List['resources_lib.Resources'], List[str], Optional[str]]:
```

Comment:
Seems the return value is getting longer. Could we also do a refactoring for it in this PR, e.g., creating a data class for the return type to make it simpler?
```
@dataclasses.dataclass
class FeasibleResourcesWithHints:
    pass
```

Reply:
Should be fixed now! Added `resource_utils.FeasibleResources`.